processors/nitf21GeospatialConverters/geographicCoordsConverter.ts

```diff
@@ -61,12 +56,12 @@ export class GeographicCoordsConverter extends BaseProcessor {
                 }
 
                 // Make call to Coordinate Conversion service to get results back in WGS84 Decimal Degrees
-                const coordinateConversionService = new CoordinateConversionService(this.executionContext.correlationId);
+                const coordinateConversionService = new CoordinateConversionService(this.executionContext.correlationId, this.logger);
```

Comment:
Should this be updated to "const coordinateConversionService: CoordinateConversionService = new …" similar to the nitf21Helper?

Reply:
Similar to the comment above: the type in cases such as these in TypeScript is inferred and unnecessary. We really only need to strong type when:

- We are initializing a variable to null to "set" later
- We are casting either to a base class or derived class
- The type could be of more than 1 type e.g. we say `doit(): string|boolean` means when we call the method we should `const result: boolean = doit()`